setup.py

```diff
@@ -30,7 +22,6 @@ def run(self):
 
 setup(
     name='pytest-localserver',
-    version=VERSION,
```

Comment:
Depending on the time line i recommend to try setup-py-update to move stuff to setup.cfg

Reply:
Yeah, I'm also in favor of migrating stuff to `setup.cfg` but I thought that's best left for a separate PR.